rocky/onboarding/templates/partials/boefje_tile_static.html

```diff
@@ -4,15 +4,15 @@
 {% for boefje in boefjes %}
     <div role="group">
         <img src="{{ boefje.tile_image }}" alt="placeholder image" />
+        <p class="plugin-title">
+            <strong>{{ boefje.name }}</strong><span class="label-plugin-type {{ boefje.type }}">{{ boefje.type|title }}</span>
+        </p>
+        <p class="plugin-description">{{ boefje.description }}</p>
         <div class="horizontal-view scan-intensity">
-            <span class="de-emphasized">{% translate "Scan level:" %}</span>
             <ul class="level-indicator {{ boefje.scan_level }}">
                 {% for i in "1234"|make_list %}<li></li>{% endfor %}
             </ul>
         </div>
-        <p>
-            <strong>{{ boefje.name }}</strong>
-        </p>
-        <p>{% translate boefje.description %}</p>
+        <span class="plugin-author-link de-emphasized"><a target="_blank" href="https://openkat.nl">OpenKAT</a></span>
```

Comment:
Same here. This template also seems to be a duplicate of plugin_tile.html?

Reply:
Yes this is the static one showing in onboarding which does not show the action buttons. Maybe I can create a no show var and pass it to the partial.